src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -641,7 +649,7 @@ ServerHandlerImp::processRequest (Port const& port,
             continue;
         }
 
-        if (method.isNull())
+        if (!jsonRPC.isMember(jss::method) || jsonRPC [jss::method].isNull())
```

Comment:
Weird formatting here too. (space before `[]`)

Reply:
This file currently has a mix of old style with significant spacing and the newer clangformatted style. Is it worth reformatting the entire file as a separate commit (that wouldn't be squashed)?